tool/tsh/proxy.go

```diff
@@ -211,7 +217,7 @@ func onProxyCommandDB(cf *CLIConf) error {
 			"database": routeToDatabase.ServiceName,
 			"type":     dbProtocolToText(routeToDatabase.Protocol),
 			"cluster":  profile.Cluster,
-			"command":  cmd.String(),
+			"command":  fmt.Sprintf("%s %s", strings.Join(cmd.Env, " "), cmd.String()),
```

Comment:
Good question. Do we have any powershell support 😅 ? If there is a way to check what shell someone is using adding if sounds simple. Otherwise, this can be a bigger task.
I think that the bigger issue is the Teleport Connect support, as currently it doesn't sets any environment variables.

Reply:
I have no idea. @smallinsky @r0mant do you have any insight on how many customers are using windows? If we have to support, maybe we can address this in a separate PR if complicated